### ISSUE TITLE
Remove parenthesis from FROM clause in Active Record

### ISSUE THREAD

**Issue by officialkirill** [bcit-ci/CodeIgniter#1257]
Using parenthesis in FROM clause doesn't do anything in SQL.
Using AR for generating queries (get_compiled_select) is really handy for combining those queries with UNION. But having ( ) in FROM clause is considered invalid syntax and breaks the query (in MySQL at least)

So the idea is to compile select queries with AR and then use them in something like $this->db->query("($sql1) UNION (sql2) UNION ($sqlx)");

To sum up: using ( ) in FROM doesn't add any functionality and removing parenthesis doesn't break anything, just adds the ability to safely use generated SQL elsewhere.
This way, we'll be able to use active record for building elaborate queries without breaking them
